.gitignore

```diff
@@ -3,3 +3,4 @@
 lib/
 node_modules
 usage.txt
+.idea/
```

Comment:
Just my two cents: We don't have to "worry" about everyone's preferred IDE, if we just accept their addition to the gitignore whenever it comes. Also, I'm guessing this won't be the last time someone tries to add `.idea/` to the gitignore. 

But I agree, either way this isn't worth worrying about lol